runtime/indent/obl.vim

```diff
@@ -0,0 +1,52 @@
+" Vim indent file
+" Language:    Oblivion Language (obl)
+" Original Creator: Kat <katisntgood@gmail.com>
+" Maintainer:  Kat <katisntgood@gmail.com>
+" Created:     01 November 2021
+" Last Change: 13 November 2022
+
```

Comment:
Likewise, could you please set `b:undo_indent` to undo the configuration changes?

See `:help undo_indent` for details and https://github.com/vim/vim/tree/master/runtime/indent for examples?